libs/blocks/quiz/utils.js

```diff
@@ -429,13 +429,21 @@ export const getAnalyticsDataForBtn = (selectedQuestion, selectedCards) => {
   return '';
 };
 
-export const getAnalyticsDataForLocalStorage = (answers) => {
+export const getAnalyticsDataForLocalStorage = (answers, umbrellaProduct, primaryProducts) => {
```

Comment:
Could you opt for a configuration object instead? `({ answers, umbrellaProduct, primaryProducts } = {})`. This would make things more manageable if a need for additional arguments arises in the future, you wouldn't need to pay attention to order or availability of all the properties

Reply:
Done. Using a config object now.